pallets/vc-management/src/tests.rs

```diff
@@ -44,6 +45,42 @@ fn request_vc_works() {
 	});
 }
 
+#[test]
+fn request_vc_with_authorised_delegatee_works() {
```

Comment:
I would name it `request_vc_13_with_authorised_delegatee_works` to indicate that we should have such assertion for every vc that ensures delegatee is a caller.

Reply:
Good point, thanks